### DISCUSSION TITLE
No scroll in recent repositories

### DISCUSSION THREAD

**Discussion by imron-mirzarasulov** [community/community#37033] [Other Feature Feedback, Questions, & Ideas] [answered]
When you click on "show more" you can't see all the projects because there is no scroll. If I add to this block style "overflow-y: auto" there're other projects which I can't see without adding own styles.

![cTCTThycQI](https://user-images.githubusercontent.com/62200534/197394623-829138b3-efe8-4c48-b149-4d71d0d9494c.png)

**Comment by ossanna16** [comment]
Hi there @web-dev404 and welcome to our community 😄. Thanks so much for reporting this!

**Comment by gunter423** [comment, marked as answer]
Hi !

There is an extension chrome  which added scroll in recent repository [https://github.com/refined-github/refined-github](https://github.com/refined-github/refined-github)

:)

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
